zktest/src/archive/test2/B85-ZK-3972-1.zul

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?import org.zkoss.zk.ui.util.*?>
+<!--
+B85-ZK-3972-1.zul
```

Comment:
Rename to B85-ZK-3972.zul, so we can test it in zktest test2 dashboard

Reply:
ok